pkg/actuators/machine/stubs.go

```diff
@@ -96,16 +96,16 @@ func stubProviderConfig() *providerconfigv1.AWSMachineProviderConfig {
 	}
 }
 
-func stubMachine() (*clusterv1.Machine, *providerconfigv1.AWSMachineProviderConfig, error) {
+func stubMachine() (*clusterv1.Machine, error) {
 	machinePc := stubProviderConfig()
 
 	codec, err := providerconfigv1.NewCodec()
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed creating codec: %v", err)
+		return nil, fmt.Errorf("failed creating codec: %v", err)
 	}
 	config, err := codec.EncodeProviderConfig(machinePc)
 	if err != nil {
-		return nil, nil, fmt.Errorf("encodeToProviderConfig failed: %v", err)
+		return nil, fmt.Errorf("encodeToProviderConfig failed: %v", err)
```

Comment:
Message is different to function called: s/encodeToProviderConfig/EncodeProviderConfig. Will help long-term grep from logs. :)